### PR TITLE
Integrate httpie

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: manala.skeleton/defaults/main.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ manala_skeleton_options:
   influxdb:              false
   dnsmasq:               false
   sqlite:                false
+  httpie:                false
```
Comment (nervo) [inline manala.skeleton/defaults/main.yml]: This is unused, and i don't think it must be used :)

I was more thinking about something like "thefuck", an always available package

File: manala.skeleton/vars/main.yml [hunk 2]
```diff
@@ -42,7 +42,8 @@ manala_skeleton_patterns:
     apt_preferences_exclusive: true
     apt_preferences: "{{
       [
-        'thefuck@manala'
+        'thefuck@manala',
+        'httpie@manala'
```
Comment (nervo) [inline manala.skeleton/vars/main.yml]: You only miss the installation of the package itself :)

**Comment by chalasr** [comment]
All fixed!

**Comment by nervo** [comment]
Nice, thx !
One more thing: the related changelogs :) We gonna talk about this tomorow, the workflow is still uncertain.

**Comment by chalasr** [comment]
changelogs updated. I'll take any rewording suggestion :)

**Pull request merged**